sw/device/tests/spi_host_flash_test_impl.c

```diff
@@ -119,17 +119,26 @@ status_t test_page_program(dif_spi_host_t *spi) {
   return OK_STATUS();
 }
 
-status_t test_page_program_quad(dif_spi_host_t *spi, uint8_t opcode) {
+status_t test_page_program_quad(dif_spi_host_t *spi, uint8_t opcode,
+                                uint8_t address_width) {
   enum {
 
     kPageSize = 256,
     kAddress = kPageSize * 10
   };
+  TRY_CHECK(address_width == 1 || address_width == 2 || address_width == 4);
+
+  dif_spi_host_width_t addr_width = address_width == 1
+                                        ? kDifSpiHostWidthStandard
+                                    : address_width == 2 ? kDifSpiHostWidthDual
+                                                         : kDifSpiHostWidthQuad;
```

Comment:
NIT: the formatting for this is very strange - how would you feel about a switch instead of the double ternary?

Reply:
You are right, done.